### PR TITLE
add dbt mesh x sl faqs

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds an faq in various locations so we can provide users with info on how to best implement dbt mesh with the SL.

needs to be reviewed by PM

### CODE REVIEW

**Review by rpourzand — APPROVED** [review]
LGTM! Might also want to tag in @jtcohen6  for a review too.

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one global project that contains your Semantic Layer configurations.
+- Then as you build your Semantic Layer, you can cross reference dbt models across your various projects or packages to create your semantic models using the [two-argument `ref` function](/reference/dbt-jinja-functions/ref#ref-project-specific-models)( `ref('project_name', 'model_name')`).
+- Your dbt Semantic Layer project serves as a global source of truth across the rest of your projects.
+
+#### Usage example 
+For example, to let's say you have a public model (`fct_orders`) that lives in the `jaffle_finance` project. As you build your semantic model, use the following syntax to ref the model:
```
Comment (nataliefiann) [inline website/snippets/_sl-dbt-mesh-faq.md]:
```suggestion
For example, let's say you have a public model (`fct_orders`) that lives in the `jaffle_finance` project. As you build your semantic model, use the following syntax to ref the model:
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for flagging this. I made one small change / fix but other than that, it's great! I've approved this for you. 

Kind Regards
Natalie

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<File name="models/metrics/semantic_model_name.yml">
+
+```yaml
+semantic_models:
+  - name: customer_orders
+    defaults:
+      agg_time_dimension: first_ordered_at
+    description: |
+      Customer grain mart that aggregates customer orders.
+    model: ref('analytics', 'fct_orders') # ref('project_name', 'model_name')
```
Comment (mirnawong1) [inline website/snippets/_sl-dbt-mesh-faq.md]:
```suggestion
    model: ref('jaffle_finance', 'fct_orders') # ref('project_name', 'model_name')
```

**Comment by mirnawong1** [comment]
Hey @jtcohen6 , just a friendly flag to double check if this faq looks ok?

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one global project that contains your Semantic Layer configurations.
+- Then as you build your Semantic Layer, you can cross reference dbt models across your various projects or packages to create your semantic models using the [two-argument `ref` function](/reference/dbt-jinja-functions/ref#ref-project-specific-models)( `ref('project_name', 'model_name')`).
```
Comment (jtcohen6) [inline website/snippets/_sl-dbt-mesh-faq.md]: I recommend also linking to:
- https://docs.getdbt.com/docs/collaborate/govern/project-dependencies

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    entities:
+      ...rest of configuration...
+    dimensions:
+      ...rest of configuration...
+    measures:
+      ...rest of configuration...
+```
+</File>
+
+Notice that in the `model` parameter, we're using the `ref` function to reference the `fct_orders` model in the `jaffle_finance` project.
```
Comment (jtcohen6) [inline website/snippets/_sl-dbt-mesh-faq.md]:
```
Notice that in the `model` parameter, we're using the `ref` function with two arguments to reference the public model `fct_orders` defined in the `jaffle_finance` project.
```

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one global project that contains your Semantic Layer configurations.
```
Comment (jtcohen6) [inline website/snippets/_sl-dbt-mesh-faq.md]: I think "global" is potentially confusing. It's a project just like other projects, but the important point is that it's "standalone" or "separate" from other (domain-aligned) projects.
```suggestion
- You have one standalone project that contains your semantic models and metrics.
```

**Review by jtcohen6 — COMMENTED** [review]
thank you @mirnawong1! sorry for the delay

cc @rpourzand - just for interest :)

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one standalone project that contains your semantic models and metrics.
+- Then as you build your Semantic Layer, you can cross reference dbt models across your various projects or packages to create your semantic models using the [two-argument `ref` function](/reference/dbt-jinja-functions/ref#ref-project-specific-models)( `ref('project_name', 'model_name')`).
```
Comment (mirnawong1) [inline website/snippets/_sl-dbt-mesh-faq.md]:
```suggestion
- Then as you build your Semantic Layer, you can [cross-reference dbt models](/docs/collaborate/govern/project-dependencies) across your various projects or packages to create your semantic models using the [two-argument `ref` function](/reference/dbt-jinja-functions/ref#ref-project-specific-models)( `ref('project_name', 'model_name')`).
```

File: website/snippets/_sl-dbt-mesh-faq.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    entities:
+      ...rest of configuration...
+    dimensions:
+      ...rest of configuration...
+    measures:
+      ...rest of configuration...
+```
+</File>
+
+Notice that in the `model` parameter, we're using the `ref` function to reference the `fct_orders` model in the `jaffle_finance` project.
```
Comment (mirnawong1) [inline website/snippets/_sl-dbt-mesh-faq.md]:
```suggestion
Notice that in the `model` parameter, we're using the `ref` function with two arguments to reference the public model `fct_orders` defined in the `jaffle_finance` project.
```

**Comment by mirnawong1** [comment]
brilliant! thanks @jtcohen6 @nataliefiann @rpourzand !

**Pull request merged**